business/openid_auth.go

```diff
@@ -722,3 +717,31 @@ func parseTimeClaim(claimValue interface{}) (int64, error) {
 
 	return parsedTime, nil
 }
+
+func createHttpClient(toUrl string) (*http.Client, error) {
+	cfg := config.Get().Auth.OpenId
+	parsedUrl, err := url.Parse(toUrl)
+
+	if err != nil {
+		return nil, err
+	}
+
+	httpTransport := &http.Transport{}
+	if cfg.InsecureSkipVerifyTLS {
+		httpTransport.TLSClientConfig = &tls.Config{
+			InsecureSkipVerify: true,
+		}
+	}
+
+	if cfg.HTTPProxy != "" || cfg.HTTPSProxy != "" {
+		proxyFunc := getProxyForUrl(parsedUrl, cfg.HTTPProxy, cfg.HTTPProxy)
```

Comment:
Looks like you "restored" the bug about using twice the HTTPProxy variable...

Reply:
That's weird, not sure what happened there.